.eslintrc.cjs

```diff
@@ -31,6 +31,7 @@ module.exports = {
       },
     ],
     'react/react-in-jsx-scope': 'off',
+    '@typescript-eslint/no-empty-function': 'off',
```

Comment:
Would you mind disabling this only for the test files?

```
"overrides": [
    {
      "files": ["src/**/*.test.ts(x)", "src/**/*.spec.ts(x)"],
      "rules": {
        '@typescript-eslint/no-empty-function': 'off',
      }
    }
  ]
```

Not 100% sure about the syntax but should be something like this

Reply:
Fixed